content/authors/neha0812/index.md

```diff
@@ -0,0 +1,47 @@
+---
+# Display name
+title: <Neha Sharma>
```

Comment:
Can you remove the angle brackets around your name? e.g.
```
title: Neha Sharma
```

Reply:
I have updated the errors. Thanks for pointing them out .